beokay.py

```diff
@@ -132,6 +132,28 @@ def control_host_bootstrap(parsed_args):
     cmd = ["kayobe", "control", "host", "bootstrap"]
     run_kayobe(parsed_args, cmd)
 
+def create_env_vars_script(parsed_args):
+    """
+    Creates an env-vars script for the kayobe environment.
+    """
+    env_vars_path = os.path.join(get_path(parsed_args), 'env-vars.sh')
+    
+    with open(env_vars_path, 'w') as f:
+        f.write("#!/bin/bash\n")
+        
+        if parsed_args.vault_password_file:
+            f.write(f"export KAYOBE_VAULT_PASSWORD=$(cat {parsed_args.vault_password_file})\n")
+        
+        f.write(f"source {get_path(parsed_args, 'venvs', 'kayobe', 'bin', 'activate')}\n")
+        f.write(f"source {get_path(parsed_args, 'src', 'kayobe-config', 'kayobe-env')}\n")
```

Comment:
Would be nice to have support for multiple environment deployments

Reply:
Ah! Good Idea